main.cpp

```diff
@@ -65,10 +61,41 @@ void variableDeclarations()
 {
     //example:
     int number = 2; //declaration of a primitive named 'number' with an initial value of '2'
+    int floor = 13;
+    int seatNum = 1;
+
+    char symb1 = 'c';
+    char symb2 = 'b';
+    char symb3 = ' ';
+
+    bool trigger = true;
+    bool triggerNext = false;
+    bool statementIs = true;
     
-    
+    float num1 = 49506;
+    float num2 = 984;
+    float num3 = 395.5;
```

Comment:
You are initialing floats with integers or doubles.  Floats have `f` at the end.
- [ ] revise